build.sbt

```diff
@@ -1,7 +1,7 @@
 val catsVersion = "2.7.0"
 
 ThisBuild / tlBaseVersion := "21.8" // TODO
-ThisBuild / tlCiReleaseBranches := Seq()
+ThisBuild / tlCiReleaseBranches := Seq("master")
```

Comment:
If/when I change the default branch name to `main`, we'll need to update this as well, right? (And then regenerate the GHA files?)

Reply:
Yes, exactly. Actually, at that point it can be omitted, since it the default setting.

Reply:
Btw, feel free to let this PR sit till the branch rename. Unless you need snapshots :)

Reply:
I updated the default branch name to `main` today, so let's omit this.

Reply:
Should be good to go!

Reply:
Thanks!